addons/portal/views/portal_templates.xml

```diff
@@ -225,27 +234,100 @@
             <div class="o_portal_my_home">
                 <div class="oe_structure" id="oe_structure_portal_my_home_1"/>
                 <div class="o_portal_docs row g-2">
-                    <div t-if="portal_alert_category_enable" class="o_portal_category row g-2 mt-3" id="portal_alert_category"/>
                     <div t-if="portal_client_category_enable" class="o_portal_category row g-2 mt-3" id="portal_client_category"/>
-                    <div t-if="portal_service_category_enable" class="o_portal_category row g-2 mt-3" id="portal_service_category"/>
-                    <div t-if="portal_vendor_category_enable" class="o_portal_category row g-2 mt-3" id="portal_vendor_category"/>
-                    <div class="o_portal_category row g-2 mt-3" id="portal_common_category">
-                        <t t-call="portal.portal_docs_entry" t-if="False"/>
-                        <t t-call="portal.portal_docs_entry">
```

Comment:
You remove the calls to `portal.portal_docs_entry` but you don't seem to remove that template definition. You should probably keep calling it inside your foreach. Allowing custom code to still use `portal.portal_docs_entry` too.

So something like

```xml
<t t-foreach="portal_entries" t-as="entry">
    <t t-call="portal.portal_docs_entry"/>
</t>
```
(either giving the full entry to the sub template like that :point_up: or specific values, like this :point_down: )
```xml
<t t-foreach="portal_entries" t-as="entry">
    <t t-call="portal.portal_docs_entry">
        <t t-set="url" t-value="entry.url"/>
        ...
        ...
    </t>
</t>
```